CHANGELOG.md

```diff
@@ -9,6 +9,30 @@ Not yet released; provisionally v2.0.0 (may change).
 Support has been added for testing against a locally running mysql docker image,
 in addition to a locally running mysql instance.
 
+### Deprecated Fields Removed From SignedLogRoot Proto
+
+*Important Note*: If this version of logserver is deployed in conjuction with
+a CTFE that's older than about 11 months it won't work properly. Make sure
+to update CTFE first.
+
+All the fields marked as deprecated in this proto have been removed. All
+the same fields are available via the TLS marshalled log root in the proto.
+Updating affected code is straightforward. For example HTTP handler code that
+needs to use these fields can do something like:
+
+```
+var currentRoot types.LogRootV1
```

Comment:
Please direct people to use the [`VerifySignedLogRoot`](https://github.com/google/trillian/blob/e6c1e8cf1ccff4929a9a5624f1a4f77c2f5bb40e/crypto/verifier.go#L33) utility functions.  Unmarshal is unsafe before the signature has been verified.

Reply:
Unsafe in what way (assuming you trust the server)?

Reply:
Clients should never trust the server ;-) - that's the whole point of being verifiable. 
If you trust the server Unmarshal is fine, but we should probably not encourage that.

Reply:
Well yes but that's how we started out with CT where it's a single system. I'll try to clarify the doc.

Reply:
Thanks

Reply:
Done.